src/pybind/mgr/cephadm/serve.py

```diff
@@ -1566,20 +1566,7 @@ async def _run_cephadm(self,
         if not self.mgr.cgroups_split:
             final_args += ['--no-cgroups-split']
 
-        if not timeout:
-            # default global timeout if no timeout was passed
-            timeout = self.mgr.default_cephadm_command_timeout
-            # put a lower bound of 60 seconds in case users
-            # accidentally set it to something unreasonable.
-            # For example if they though it was in minutes
-            # rather than seconds
-            if timeout < 60:
-                self.log.info(f'Found default timeout set to {timeout}. Instead trying minimum of 60.')
-                timeout = 60
-            # subtract a small amount to give this timeout
-            # in the binary a chance to actually happen over
-            # the asyncio based timeout in the mgr module
-            timeout -= 5
+        timeout = self._set_cephadm_command_timeout(timeout)
```

Comment:
I think probably is better to substract the `-5` here instead of having it as part of the method logic. This way the UT would be cleaner and clear as the value returns is what we expect without any tweaks. WDYT?